.gitignore

```diff
@@ -0,0 +1,334 @@
+!*.[Cc]ache/
+
+# Others
+ClientBin/
+~$*
+*~
+*.dbmdl
+*.dbproj.schemaview
+*.jfm
+*.pfx
+*.publishsettings
+orleans.codegen.cs
+
+# Since there are multiple workflows, uncomment next line to ignore bower_components
+# (https://github.com/github/gitignore/pull/1529#issuecomment-104372622)
+#bower_components/
+
+# RIA/Silverlight projects
+Generated_Code/
+
+# Backup & report files from converting an old project file
+# to a newer Visual Studio version. Backup files are not needed,
+# because we have git ;-)
+_UpgradeReport_Files/
+Backup*/
+UpgradeLog*.XML
+UpgradeLog*.htm
+
+# SQL Server files
+*.mdf
+*.ldf
+*.ndf
+
+# Business Intelligence projects
+*.rdl.data
+*.bim.layout
+*.bim_*.settings
+
+# Microsoft Fakes
+FakesAssemblies/
+
+# GhostDoc plugin setting file
+*.GhostDoc.xml
+
+# Node.js Tools for Visual Studio
+.ntvs_analysis.dat
+node_modules/
+
+# Typescript v1 declaration files
+typings/
+
+# Visual Studio 6 build log
+*.plg
+
+# Visual Studio 6 workspace options file
+*.opt
+
+# Visual Studio 6 auto-generated workspace file (contains which files were open etc.)
+*.vbw
+
+# Visual Studio LightSwitch build output
+**/*.HTMLClient/GeneratedArtifacts
+**/*.DesktopClient/GeneratedArtifacts
+**/*.DesktopClient/ModelManifest.xml
+**/*.Server/GeneratedArtifacts
+**/*.Server/ModelManifest.xml
+_Pvt_Extensions
+
+# Paket dependency manager
+.paket/paket.exe
+paket-files/
+
+# FAKE - F# Make
+.fake/
+
+# JetBrains Rider
+.idea/
+*.iml
+
+# Maven target
+**/java/**/target/
+
+# CodeRush
+.cr/
+
+# Python Tools for Visual Studio (PTVS)
+__pycache__/
+*.pyc
+
+# Cake - Uncomment if you are using it
+# tools/**
+# !tools/packages.config
+
+# Telerik's JustMock configuration file
+*.jmconfig
+
+# BizTalk build output
+*.btp.cs
+*.btm.cs
+*.odx.cs
+*.xsd.cs
+
+# Unity project
+quickstart/csharp/unity/*/[Ll]ibrary/
+quickstart/csharp/unity/*/[Tt]emp/
+quickstart/csharp/unity/*/[Oo]bj/
+quickstart/csharp/unity/*/[Bb]uild/
+quickstart/csharp/unity/*/[Bb]uilds/
+quickstart/csharp/unity/*/[Ll]ogs/
+quickstart/csharp/unity/*/[Uu]ser[Ss]ettings/
+
+quickstart/csharp/unity/*/*.csproj
+quickstart/csharp/unity/*/*.sln
+quickstart/csharp/unity/*/*.apk
+
+quickstart/csharp/unity/*/Assets/SpeechSDK/
+
+# Linux executables
+quickstart/cpp-linux/helloworld
+samples/cpp/windows/console/samples/sample
+
+# macOS executables
+quickstart/cpp-macos/helloworld
+quickstart/text-to-speech/cpp-macos/helloworld
+
+# synthesized audio
+quickstart/**/text-to-speech/*.wav
+
+# compiled JavaScript files
+quickstart/javascript/**/microsoft.cognitiveservices.speech.sdk.bundle*.js*
+samples/js/**/microsoft.cognitiveservices.speech.sdk.bundle*.js*
+
+# generated pods directory and xcode workspace
+**/swift/**/Pods/
+**/swift/**/Podfile.lock
+**/swift/**/*.xcworkspace/
+**/objectivec/**/Pods/
+**/objectivec/**/Podfile.lock
+**/objectivec/**/*.xcworkspace/
+
+# iOS framwork
+samples/objective-c/ios/MicrosoftCognitiveServicesSpeech.framework/
```

Comment:
please remove unnecessary lines

Reply:
removed ios part and keep the Visual Studio and Java part.

Reply:
please just keep C# and python part because we only have sample code for these 2 languages.